internal/kafka/command_topic_produce.go

```diff
@@ -632,3 +646,26 @@ func setSchemaPathRef(schemaString srsdk.SchemaString, dir, subject string, sche
 	}
 	return tempStorePath, referencePathMap, nil
 }
+
+func parseHeaders(headers []string, delimiter string) ([]ckafka.Header, error) {
+	if len(headers) == 0 {
+		return nil, fmt.Errorf(missingHeadersErrorMsg, delimiter)
+	}
```

Comment:
It seems like we might hit this case if the `--headers` flag is not passed, since the default value is `nil`?

Reply:
Nice catch, I didn't thought about this case. You are right it will hit this case by default. I have removed the condition as it seems not useful because the `--headers` should be optional and can be empty by default.